### PR TITLE
[4509][3.1.x] RTE wrap for long lines of code or text

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4509

### CODE REVIEW

File: static-assets/modules/editors/tinymce/v5/tinymce/plugins/acecode/plugin.js
Comment (rart) [inline static-assets/modules/editors/tinymce/v5/tinymce/plugins/acecode/plugin.js]: @russdanner @sumerjabri I think this is a preference and I'm not sure which should be the default.
Some people don't like wraps and looks like we're changing behaviour that's been there for a long time.

We have some ace configuration one of the config xmls (don't recall which at the moment). Shall we add this one there?

Comment (sumerjabri) [reply static-assets/modules/editors/tinymce/v5/tinymce/plugins/acecode/plugin.js]: I think we add the option/config anyway.

@russdanner I'm in favor of the default being `wrap` because that's my personal preference, however, it's your call what should the default be.

I would have loved to see a button on ACE letting me choose in real-time.

**Comment by jvega190** [comment]
Now it is configurable from the rte (tinymce 5) configuration file. I'll update the bps to include it. Default was set to false (current behavior), waiting for Russ' response.
@sumerjabri should the configuration update be handled by the Upgrade Manager too?

**Review by rart — APPROVED** [review]
We could merge and change the default if Russ comes back requesting the change of default.

**Pull request merged**
